cloudsim-plus-examples/src/main/java/org/cloudsimplus/examples/power/PowerExample.java

```diff
@@ -354,4 +352,8 @@ private Host createPowerHost() {
         return list;
     }
 
+    static double wattsSecToKWattsHour(final double power) {
```

Comment:
The current implementations doesn't since I found it a little arbitrary to include this utility function on some random power class. One could as well search it under `PowerMeter` or `PowerMeasurement` or some `PowerModel`. Don't think we need to include this in the core classes, I'm of course absolutely fine with adding it back if you prefer that though :)

Reply:
The thing is that this method can be used in other examples and is useful for other researchers.
Moving it to a specific example will end up causing code duplication.
Maybe we could find a better place for it, such as inside the Conversion class.

Reply:
Putting it in the Conversion class makes a lot of sense! Did not know it existed :)